src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
Okay. I added a commit that enables ``WIN64EXCEPTIONS`` and removes unnecessary stubs from ``excepcpu.h`` and ``excepx86.cpp``.